bundle/manifests/netobserv-operator.clusterserviceversion.yaml

```diff
@@ -351,7 +351,7 @@ metadata:
     capabilities: Seamless Upgrades
     categories: Monitoring
     console.openshift.io/plugins: '["netobserv-plugin"]'
-    containerImage: quay.io/netobserv/network-observability-operator:1.0.3
+    containerImage: quay.io/meth/network-observability-operator:main
```

Comment:
you can run `make update-bundle` to avoid this :smile_cat: 
https://github.com/netobserv/network-observability-operator/blob/84c7fe8057a58d9d5d23c2ea3ab92d83499acc6c/Makefile#L325-L327

Reply:
fixed